### PR TITLE
create outline for journal week 3

### PR DESCRIPTION
**Pull request by drumslayert**

### CODE REVIEW

**Comment by s4dmw** [comment]
good start! :+1:

**Comment by jeremyrist** [comment]
MVC all the way! :+1:

**Comment by karjac14** [comment]
great start, will look forward for the expanded version. :+1:

File: tutorial-week-3.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+###How to color your font!
+
+ * what font?
+ * oh that font!
+ * what color? 
+ * oh so many colors!
+ * the color purple! (such a great movie too)
```
Comment (kellymurray) [inline tutorial-week-3.md:7]: such a great movie + book!

**Pull request merged**